tests/test_image.py

```diff
@@ -53,20 +54,40 @@ def testImageSqrt(self):
         self.assertTrue(np.allclose(self.ims.sqrt().asnumpy(), np.sqrt(self.ims_np)))
 
     def testImageTranspose(self):
+        # test method and abbreviation
         self.assertTrue(
             np.allclose(
-                self.im.flip_axes().asnumpy(), np.transpose(self.im_np, (0, 2, 1))
+                self.im.T.asnumpy(),
+                self.im.transpose().asnumpy(),
+                np.transpose(self.im_np, (0, 2, 1)),
```

Comment:
Same bug as below. The third term to allclose will be considered `rtol` in the formula `absolute(a - b) <= (atol + rtol * absolute(b))`.